galata/test/galata/fixture.spec.ts

```diff
@@ -161,8 +161,8 @@ test.describe('terminals', () => {
 
 test.describe('tmpPath', () => {
   test('should return an unique test folder', ({ tmpPath }) => {
-    expect(tmpPath).toEqual(
-      'test-galata-fixture-tmpPath-should-return-an-unique-test-folder-galata'
+    expect(tmpPath).toMatch(
+      /test-galata-fixture-tmpPat-\w{5}-eturn-an-unique-test-folder-galata/
```

Comment:
Maybe we can add a quick comment to remember why we need this regex? Probably because the text is truncated?

Reply:
Comment added in [2fdea68](https://github.com/jupyterlab/jupyterlab/pull/15621/commits/2fdea684cad0b806f6b36b998c055f96cd68ceef)